pysrc/juliacall/__init__.py

```diff
@@ -129,7 +129,7 @@ def args_from_config():
     CONFIG['opt_sysimage'] = sysimg = path_option('sysimage', check_exists=True)[0]
     CONFIG['opt_threads'] = int_option('threads', accept_auto=True)[0]
     CONFIG['opt_warn_overwrite'] = choice('warn_overwrite', ['yes', 'no'])[0]
-    CONFIG['opt_handle_signals'] = 'no'
+    CONFIG['opt_handle_signals'] = choice('handle_signals', ['yes', 'no'], default='no')[0]
```

Comment:
Can we default to `yes`?  Defaulting to a behaviour which only causes crashes for backward compatibility doesn't seem very useful.

Reply:
> Can we default to `yes`? Defaulting to a behaviour which only causes crashes for backward compatibility doesn't seem very useful.

I agree, that's probably better to default to `yes`, but I left off for now as it is a behavioral change. Just posted a comment on this thread too, let's see what the author thinks